etc/otel-jmx.config.yaml

```diff
@@ -43,31 +63,95 @@ rules:
        type: gauge
        desc: total records consumed
        unit: '{records}'
- - bean: kafka.streams:type=stream-task-metrics,thread-id=*,task-id=*
+
+ # Kafka Streams thread metrics
+ - bean: kafka.streams:type=stream-thread-metrics,thread-id=*
    metricAttribute:
      thread: param(thread-id)
-     task: param(task-id)
    mapping:
      process-total:
-       metric: kafka.streams.task.process.total
+       metric: kafka.streams.thread.process.total
        type: gauge
        desc: total records processed
        unit: '{records}'
- - bean: kafka.streams:type=stream-thread-metrics,thread-id=*
-   metricAttribute:
-     thread: param(thread-id)
-   mapping:
+     process-rate:
+       metric: kafka.streams.thread.process.rate
+       type: gauge
+       desc: rate of records processed
+       unit: '{records-per-second}'
+     process-ratio:
+       metric: kafka.streams.thread.process.ratio
```

Comment:
Left all the ratios unit-less but open to suggestions

Reply:
that's fine - I think the units are just informational anyway